web/tasks/CommonGen.md

```diff
@@ -0,0 +1,208 @@
+
+A screenshot of the interface is provided in Figure 7 of the [Appendix](https://arxiv.org/pdf/1911.03705v3.pdf).
+
+#### Who are the source language producers?
+
+The original language of this dataset was created by Amazon Mechanical Turk workers.
+
+During the data collection, workers who provided rationales that were too short, failed to have good coverage of the input in their sentences, or workers whose output had a high perplexity under a GPT-2 model were disqualified from the pool and replaced with newcomers.
+
+No demographic information is provided
+
+### Annotations
+
+The dataset does not contain any additional annotations,
+
+#### Annotation process
+
+[N/A]
+
+#### Who are the annotators?
+
+[N/A]
+
+### Personal and Sensitive Information
+
+The concepts are restricted to verbs, adjectives, and common nouns, and no personal information is given in the captions.
+
+## Changes to the Original Dataset for GEM
+
+If the originally published dataset was modified in any way for GEM, please record the changes here. These could include data cleaning, exclusion of certain languages, changes to the data splits, additional challenge examples, among others.
+
+## Considerations for Using the Data
+
+### Social Impact of the Dataset
+
+The task is presented as a stepping stone towards building models that achieve more human-like text generation. Progress in this research direction opens the way for strong quality of life improvements through applications such as virtual assistants or automatic image captioning, while also raising concern of misuse especially by agents who want to hide that a text is machine generated.
+
+### Impact on Underserved Communities
+
+The dataset is in English, a language with an abundance of existing resources.
+
+### Discussion of Biases
+
+The dataset is created using data from image captioning systems and might inherit some of the social biases represented therein (see e.g. [Tang et al. 2020](https://arxiv.org/abs/2006.08315)).
+
+Another related concern is the exposure bias introduced by the initial selection of pictures and video, which are likely to over-represent situations that are common in the US at the expense of other parts of the world (Flickr, for example, is a US-based company founded in Canada). For more discussion of the potential impacts of exposure bias, see e.g. [The Social Impact of Natural Language Processing](https://www.aclweb.org/anthology/P16-2096.pdf).
+
+The use of GPT-2 to validate development ant test sentences [might be cause for similar concern](https://www.aclweb.org/anthology/D19-1339.pdf), but we do note that the authors only use the model to discount very high perplexity sequences which is less likely to surface those biases.
+
+### Other Known Limitations
+
+The language in the development and test set is crowdsourced, which means that it was written by workers whose main goal was speed. This is likely to impact the quality and variety of the targets.
```

Comment:
Maybe also note that the population of turkers is not identically distributed as the overall population which may lead to different representation of situations or underlying expectations of what these situations are.